manifests/s/Seedy-Threepio/Theseus2000/3.6C/Seedy-Threepio.Theseus2000.locale.en-US.yaml

```
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.2.6.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: Seedy-Threepio.Theseus2000
+PackageVersion: 3.6C
+PackageLocale: en-US
+Publisher: Seedy-Threepio

```

```suggestion
Publisher: SeedyThreeSixty
```
To match control panel